wiki/_README.md

```diff
@@ -0,0 +1,53 @@
+_Certified fresh 230808 by Graham Johnson._
+
+# Navigating the docs
+
+For a complete list of entries, see our [Table of Contents](_TOC.md)
+
+# Updating the docs: How & when
+
+Whenever a code change PR breaks (or makes obsolete) existing documentation, the relevant entry should be updated. This docs update should be included within the PR itself, as part of the PR—just as one would update unit tests alongside a breaking code change.
+
+Similarly, whenever a PR adds codebase functionality where either:
+
+1. accompanying documentation has been explicitly requested 
+2. the implementing engineer believes the new functionality will not be easily legible to or extendable by future engineers
+
+...then that PR should include a new documentation entry as part of its purview.
+
+At least for the duration of 2023, as we implement this new system, any PR that updates or contributes to our documentation wiki should tag @gdjohnson (the documentarian) on Github.
+
+# Tracking documentation status
+
+An important part of maintaining a working documentation system involves flagging (1) areas in need of documentation (2) areas whose documentation has gone stale.
+
+For now, the best system is notifying the archivist, Graham Johnson, on Slack or Github (@gdjohnson). Down the line, a more structured system may be worth implementing.
+
+## "Certified fresh" datestamps
+
+Moving forward, our goal is to keep a “Certified fresh” datestamp on all wiki entries. This timestamp is accompanied by an optional note and the name of the certifier. For consistency, dates should take the format YYMMDD.
+
+Example certification:
+> _Certified fresh 230731 by Graham Johnson._
```

Comment:
Should we use @s or full names for these tags?

Also, I'm wondering if there's a way to use some sort of embedded header block to track this metadata about the document in question, but this is definitely a good start.

Reply:
Re: @-s, I was torn—I had noticed some confusion among team members as to other team members' GitHub handles, and the most important thing to me seemed like the ability to Slack DM the DRI/"certifier." This inclined me slightly toward full names, since new hires & temporary contractors might not immediately realize e.g. CowMuon is Forest. 

Let me look into docs metadata tracking solutions some more, I'll make a not in my to-dos.